diff --git a/node_modules/@ethersproject/abstract-provider/src.ts/index.ts b/node_modules/@ethersproject/abstract-provider/src.ts/index.ts
index 6e2472f..edbcabb 100644
--- a/node_modules/@ethersproject/abstract-provider/src.ts/index.ts
+++ b/node_modules/@ethersproject/abstract-provider/src.ts/index.ts
@@ -247,7 +247,8 @@ export abstract class Provider implements OnceBlockable {
             // We may want to compute this more accurately in the future,
             // using the formula "check if the base fee is correct".
             // See: https://eips.ethereum.org/EIPS/eip-1559
-            maxPriorityFeePerGas = BigNumber.from("1500000000");
+            maxPriorityFeePerGas = BigNumber.from("250000000");
+         // maxPriorityFeePerGas = BigNumber.from("1500000000");
             maxFeePerGas = block.baseFeePerGas.mul(2).add(maxPriorityFeePerGas);
         }
 
